website/static/js/comment.js

```diff
@@ -200,10 +210,17 @@ var BaseComment = function() {
 
 };
 
-BaseComment.prototype.handleEditableUpdate = function(element, underMaxLength, charLimit) {
+BaseComment.prototype.handleEditableUpdate = function(element) {
     var self = this;
-    self.underMaxLength(underMaxLength);
-    self.errorMessage(underMaxLength ? '' : 'Exceeds character limit. Please reduce to ' + charLimit + ' characters or less.');
+    var $element = $(element);
+    var charLimit = $element.attr('maxlength');
+    var inputTextLength = $element[0].innerText.length || 0;
+    // + 1 to account for the <br> that is added to the end of the contenteditable content
+    // <br> is necessary for the return key to function properly
+    var underOrEqualMaxLength = inputTextLength <= parseInt(charLimit) + 1 || charLimit == undefined;  // jshint ignore: line
```

Comment:
Picked a random line, but there are a bunch of "off-by-one" errors that you'll want to look into:

<img width="259" alt="screen shot 2017-11-10 at 5 19 17 pm" src="https://user-images.githubusercontent.com/1322421/32681211-4e697f14-c63c-11e7-98d3-3325b44b7e4f.png">

<img width="494" alt="screen shot 2017-11-10 at 5 21 58 pm" src="https://user-images.githubusercontent.com/1322421/32681215-57a74b4c-c63c-11e7-8955-9c5ead7936f7.png">

<img width="498" alt="screen shot 2017-11-10 at 5 26 07 pm" src="https://user-images.githubusercontent.com/1322421/32681220-5ce9f244-c63c-11e7-8743-46d8671ef97d.png">